Monika After Story/game/script-topics.rpy

```diff
@@ -38,7 +38,24 @@ init 11 python:
 
 #BEGIN ORIGINAL TOPICS
 
+
 #Use this topic as an example
+init 5 python:
+    # List of keywords for the topic.
+    for key in ['film''movie']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_movie') #id
+
+label monika_movie:
+    m 1b "Do you want to watch a movie with me?!" 
+    m 1q"..."
+    m 1g "jeez, I can't go to the cinema with you."
+    m 1o"..."
+    m 1k "but I gues you can put the movie in the computer"
```

Comment:
Misspelled word.

Reply:
Also consider rewording this sentence sounds weird.

Reply:
ok, can you told me how to edit this, its my firs time so..
and my native lenguage is spanish, so i am not the best in english

Reply:
nevermind